{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2020-23026",
      "ASSIGNER": "cve@mitre.org"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-476"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://fossies.org/linux/privat/old/dhrystone-2.1.tar.gz/dhry_1.c",
          "name": "https://fossies.org/linux/privat/old/dhrystone-2.1.tar.gz/dhry_1.c",
          "refsource": "MISC",
          "tags": [
            "Third Party Advisory"
          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "A NULL pointer dereference in the main() function dhry_1.c of dhrystone 2.1 causes a denial of service (DoS)."
        }
      ]
    }
  },
  "configurations": {
    "nodes": [
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:dhrystone_project:dhrystone:2.1:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "PARTIAL",
        "baseScore": 5
      },
      "severity": "MEDIUM",
      "exploitabilityScore": 10,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2022-01-03T20:15Z",
  "lastModifiedDate": "2022-01-08T02:48Z"
}